build_modules/pubspec.yaml

```diff
@@ -7,7 +7,7 @@ repository: https://github.com/dart-lang/build/tree/master/build_modules
 resolution: workspace
 
 environment:
-  sdk: '>=3.5.0 <3.7.0-z'
+  sdk: '>=3.7.0-82.0.dev <3.8.0-z'
```

Comment:
Fwiw I think we can drop the `-z` off the `<3.8.0-z` now as well. That was a short term thing to allow 3.7 dev releases but not stable.